EIPS/eip-6260.md

```diff
@@ -0,0 +1,290 @@
+---
+eip: 6260
+title: Buyable NFT tokens on-Chain & Royalties
+description: Allowing tokens to be buyable and enforce royalties directly on-Chain
+author: Aubay <blockchain-team@aubay.com>, BOCA Jeabby (@bjeabby1507), EL MERSHATI Laith (@lth-elm), KEMP Elia (@eliakemp)
+discussions-to: https://ethereum-magicians.org/t/eip-6260-buyable-nft-tokens-on-chain-and-royalties/12443
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-17
+requires: 721
+---
+
+## Abstract
+
+This EIP allows an [EIP-721](./eip-721.md) token to become buyable and to specifically enforce royalties as a percentage, directly on-chain, without entrusting a third-party. A decentralized marketplace can be built around this interface.
```

Comment:
Since collecting royalties on-chain is... difficult, you should include a terse technical summary of _how_ this proposal achieves that goal.